cypress/support/step_definitions/index.ts

```diff
@@ -24,13 +24,20 @@ const YAML = {
 
 let env = {}
 let selectors: Record<string, string> = {}
+let localStorage: Set<string>
 const client = useClient()
 Before(() => {
   client.reset()
   env = {}
   selectors = {}
+  localStorage = new Set()
   useServer()
 })
+After(() => {
+  Array.from(localStorage).forEach((key) => {
+    window.localStorage.removeItem(key)
+  })
```

Comment:
can't we just do `clear()`?

```suggestion
  window.localStorage.clear()
```

Reply:
We only want to remove the keys that are used in the tests, not the entire `localStorage`. I might have other values in localStorage that I want to keep.

Ideally we would save previous values that we might overwrite, and then put those values back in at the end of the tests, but not super important as yet